src/languages/en.js

```diff
@@ -30,6 +30,8 @@ export default {
         lastName: 'Last name',
         phone: 'Phone',
         phoneNumber: 'Phone number',
+        phonePlaceholder: '(xxx)xxx-xxxx',
+        errorMessageInvalidPhone: 'If you\'re outside the US please include your country code, eg. +447782339811',
```

Comment:
Should there be a comma at the end?
```suggestion
        errorMessageInvalidPhone: 'If you\'re outside the US please include your country code eg. +447782339811',
```

Reply:
Will ask on slack